project.mml

```diff
@@ -1427,6 +1427,7 @@ Layer:
             tags->'power_source' as power_source,
             tags->'icao' as icao,
             tags->'iata' as iata,
+            tags->'parking' as parking,
```

Comment:
What does another column do to the number of combinations, and does it cause problems with a combinatorial explosion?

Reply:
I have no deeper understanding of SQL in layers - I just try to make it work and look similar to other features. Some guidelines or tutorial for adding things would be welcome.

What you think should be done here?

Reply:
https://vimeo.com/68093876 talks about the problem starting at 10:52-14:21. The best way to check this isn't a problem here is to compare the line count of the XML output. If it increases by a dozen lines, that's fine. If it increases by 10000 lines, it's a problem.

I **suspect** this is okay, but I'm not 100% sure. If you added something that styled **all** objects with some parking value then it would definitely be a problem.

Reply:
Most of the code I have ever added to osm-carto was some SVG XML for icons, so that's probably not the case for me.

I understand the combinatorial explosion principle, my problem is rather what you can see at 16:40. Some common layers are so complex, that they don't even fit on the screen, so it's hard to know which visual section is needed and how should it look like. 

I refer to "visual sections", because with so many elements it's hard to see the logical structure and know what is nested inside something other - even with all the indents. Maybe putting some of the lists into variables or including them from other files would help, if it's possible?

Reply:
Where can I find those generated XML files, in particular having a setup closely following
https://github.com/gravitystorm/openstreetmap-carto/blob/master/DOCKER.md
?